appengine-mail/src/test/java/com/arcbees/appengine/mail/MailBuilderFromTest.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2013 ArcBees Inc.
- *
+ * <p/>
```

Comment:
Remove the `<p/>`s and remove a star from line 1 to prevent the formatter from messing up